dockerfiles/build-extension/Dockerfile_fpm

```diff
@@ -0,0 +1,23 @@
+# Copied from:
```

Comment:
This file is (roughly) a clone of the patch we already have in use, moved here from our docker-library repo.

Reply:
Since we've been consolidating all the Dockerfiles to https://github.com/DataDog/dd-trace-ci/tree/master/php, can we move these Dockerfiles there too? I think we started a conversion about this a while ago but we didn't come to any official conclusion yet, so maybe it's time to strike that conversation up again. :D

Reply:
Most recently I think we talked about bringing them back into _this_ repo, didn't we?